dataload/import_to_elasticsearch.py

```diff
@@ -483,6 +520,106 @@ def to_band(value, bins, labels):
         previous_value = b
 
 
+def update_doc_with_other_locations(grant):
+    """ This flattens out some embedded data for easier indexing """
+
+    # Prior versions of additional_data may not have this field
+    # or if locationLookup failed entirely for this grant
+    if not grant["additional_data"].get("locationLookup"):
```

Comment:
is `grant["additional_data"]` always guaranteed to be set and the correct type at this point, or does this need a try/except?

Reply:
Yeah that should be covered by the earlier check on grant import, if it doesn't exist it will create an empty `dict`

https://github.com/ThreeSixtyGiving/grantnav/blob/live/dataload/import_to_elasticsearch.py#L420